### DISCUSSION TITLE
Change description

### DISCUSSION THREAD

**Discussion by nick-gehring** [Shopify/function-examples#387] [Feedback: Delivery Customization API]
I (and more importantly my client) would love it if we could change a shipping rate description like we can rename a shipping title. In our case, we just like to blank it so customers do not get expectations of a specific delivery date.

**Comment by gil--** [comment]
I noticed this is possible with custom rates today would be great if we could set on-demand:
![CleanShot 2024-02-19 at 11 51 27@2x](https://github.com/Shopify/function-examples/assets/3484527/76a712af-94f1-4b7d-b18d-00e6f3a0e7e6)

![CleanShot 2024-02-27 at 09 43 13@2x](https://github.com/Shopify/function-examples/assets/3484527/519ecbd6-9eaa-4f6a-b38d-01a2ab8df1af)
